datascience/tables.py

```diff
@@ -243,26 +243,29 @@ def values(self, label):
 
         ``__getitem__`` is aliased to this, so bracket notation can be used:
 
+        >>> letter = ['a', 'b', 'c', 'z']
+        >>> count = [9, 3, 3, 1]
+        >>> points = [1, 2, 2, 10]
+        >>> t = Table([letter, count, points], ['letter', 'count', 'points'])
         >>> t.values('letter') == t['letter']
-        True
+        array([ True,  True,  True,  True], dtype=bool)
```

Comment:
Right, I forgot about this behavior. Do you think this is confusing if all we're trying to show is that `t.values` is the same as `t['letter']`?